src/site/layouts/step_by_step.drupal.liquid

```diff
@@ -0,0 +1,71 @@
+{% include "src/site/includes/header.html" with drupalTags = true %}
+
+{% include "src/site/includes/alerts.drupal.liquid" %}
+{% include "src/site/includes/preview-edit.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+
+<div id="content" class="interior">
+  <main class="va-l-detail-page">
+    <div class="usa-grid usa-grid-full">
+      <div class="usa-width-three-fourths">
+        {% if !entityPublished %}
+          <div class="usa-alert usa-alert-info">
+            <div class="usa-alert-body">
+              <p class="usa-alert-text">You are viewing a draft.</p>
+            </div>
+          </div>
+        {% endif %}
+
+        <article class="usa-content">
+          <!-- Title -->
+          <h1>{{ title }}</h1>
+
+          <!-- Intro -->
+          <p>{{ fieldIntroTextLimitedHtml.processed }}</p>
+
+          <!-- Alert -->
+          {% if fieldAlertSingle %}
+            {% include "src/site/paragraphs/alert_single.drupal.liquid" with entity = fieldAlertSingle.entity %}
+          {% endif %}
+
+          <!-- Buttons -->
+          {% for fieldButton in fieldButtons  %}
+            {% include "src/site/paragraphs/button.drupal.liquid" with entity = fieldButton.entity %}
+          {% endfor %}
+
+          <!-- Steps -->
+          <h2>Step-by-step</h2>
```

Comment:
Should there be a property for  this? Seems weird to have `Step-by-step` hardcoded

Reply:
Was just talking to @andaleliz about this, it wasn't clear what value we would put here so I just hardcoded it for now

Reply:
Might want to post in #learning-center-mvp and check w/ Ryan and/or Jen.  This would be good to capture in the template requirements